### ISSUE TITLE
Certificate Revocation?

### ISSUE THREAD

**Issue by wallabyies** [sjkp/letsencrypt-siteextension#47]
Is there a mechanism provided to revoke certificates?

I believe Let's Encrypt can revoke through the ACME client if one has the private key.   Is that available for export in Azure Web Apps?

[Let's Encrypt Documentation](https://letsencrypt.org/getting-started/) on Revocation:

Revoking a Certificate
The following command can be used to revoke a particular certificate.

$ certbot revoke --cert-path example-cert.pem

**Comment by sjkp** [comment]
The private key, is on the disk of the azure web site, so yes it can be downloaded if you want. 

I dont know how the revocation works or if it is possible.

**Comment by wallabyies** [comment]
I've check the deployment folder, but do not see anything related to the cert or key.    

Any specifics on where the key is stored would be very helpful.   Thank you!

**Comment by sjkp** [comment]
D:\home\SiteExtensions\letsencrypt\config\

**Comment by wallabyies** [comment]
I confirmed I was able to successfully revoke a cert acquired via this site extension. Steps I used:

1. Installed [certbot](https://certbot.eff.org/) on a linux machine.

2. Downloaded the private key and public certificate in pem format via ftps from my web app in Azure.   It was located here:  /SiteExtensions/letsencrypt/config/httpsacme-v01.api.letsencrypt.org/

3. Revoke with cerbot client:  ./certbot-auto revoke --cert-path public.pem --key-path private.pem
